components/camel-huaweicloud-smn/src/test/resources/log4j2.properties

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
We need to add the Apache License Header here.

Reply:
Thanks for pointing. License headers added.